pom.xml

```diff
@@ -174,8 +174,8 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
 
     <!-- Maven protoc compiler -->
     <protobuf-maven-plugin.version>0.5.1</protobuf-maven-plugin.version>
-    <protobuf-compile.version>3.5.0</protobuf-compile.version>
-    <grpc.version>1.10.0</grpc.version>
+    <datanode.protobuf-compile.version>3.10.0</datanode.protobuf-compile.version>
```

Comment:
is this version that is used in Hadoop trunk?

Reply:
Hadoop 3.2 uses 3.7 https://github.com/apache/hadoop/blob/7c869b4df2503555de6a0e376aab4a98c655fc94/hadoop-project/pom.xml#L88